listenbrainz_spark/recommendations/train_models.py

```diff
@@ -50,6 +52,27 @@ def compute_rmse(model, data, n):
       .values()
     return sqrt(predictionsAndRatings.map(lambda x: (x[0] - x[1]) ** 2).reduce(add) / float(n))
 
+def generate_model_id():
+    """ Generate best model id.
+    """
+    return '{}-{}'.format(config.MODEL_ID_PREFIX, uuid.uuid4())
+
+def get_dataframe_id(dataframe_metadata_df, best_model_metadata):
+    """ Get dataframe id of datafsets on which model is trained from dataframe metadata.
```

Comment:
datafsets

Reply:
And there are still a number of "best_model_metadata" references in the code/docstrings, which I find confusing. Can you please clean those up?

Reply:
> And there are still a number of "best_model_metadata" references in the code/docstrings, which I find confusing.

which references?  I guess those references are needed. Can you please point some so that I can understand better?